.github/workflows/check-pr.yml

```diff
@@ -54,3 +54,36 @@ jobs:
     with:
       stack-name: ${{ needs.deploy.outputs.stack-name }}
       aws-region: ${{ needs.deploy.outputs.aws-region }}
+
+  eventbridge-abandon-tests:
```

Comment:
Can I ask why we've implemented these separately rather than just running all the eventbridge tests at once like we do with the step function and api gateway tests?

Reply:
I think it's because these tests are independent so we can take advantage of concurrency provided by GH?

Reply:
Cool sounds like a good reason!